### DISCUSSION TITLE
Manage focus to the next "TextField Control"

### DISCUSSION THREAD

**Discussion by nastiliano** [flet-dev/flet#3703] [General]
Hello everyone,

I'd like to know if is possible force the focus to the next "TextField Control", like we press the "tab" key.

I'd like to press "enter" key (event: on_submit) and move the focus to the next natural "Control".

Thank you very much.

Best regards.

**Comment by ndonkoHenri** [comment]
Some controls have `.focus()` method which you can use so that they get focused.

**Reply by nastiliano** [reply]
Hello!

Yes, thank you, I know, but I don't know how to do it "on_submit" method.

For example:

    self.tf = ft.TextField(autofocus = True ,
                                   capitalization = "CHARACTERS" ,
                                   data = "ft" ,
                                   error_text = "" ,
                                   input_filter = ft.InputFilter(allow = True , regex_string = r"\w") ,
                                   label = "FT" ,
                                   max_length = 9 ,
                                   on_submit = ????????  NEXT CONTROL.focus() ????????????,
                                   width = 400
                                  )
I don't know how to "point" to the "next" Control

Thank you.

**Reply by ndonkoHenri** [reply]
You can do` lambda e: yourcontrol.focus()`

**Reply by nastiliano** [reply]
Yes, I know thanks.
But I want a generic instruction to put the focus on the next Control like when you press Tab key. 
I want to put the focus like pressing Tab key, not on a specific Control. Or I want an instruction to know the focus "index"

**Comment by japr99** [comment]
cantidad = ft.TextField(
        label=None,
        width=70,
        height=32,
        text_size=16,
        autofocus=True,
        text_align=ft.TextAlign.LEFT,
        text_vertical_align=ft.VerticalAlignment.CENTER,
        content_padding=ft.Padding(8,0,0,0),
        on_blur=lambda e: can_hojas_tal.focus()
    )


    # Campo de cantidad de hojas por talonario
    can_hojas_tal = ft.TextField(label=None, width=70, height=32, text_size=16, text_align=ft.TextAlign.LEFT, text_vertical_align=ft.VerticalAlignment.CENTER, content_padding=ft.Padding(8,0,0,0))
    
    Puedes ir de un campo a otro con
    
     on_blur=lambda e: campo.focus()
